torchinductor/scheduler.py

```diff
@@ -4,12 +4,17 @@
 import functools
 import itertools
 import logging
+import os
 from typing import Any
 from typing import Dict
 from typing import List
 
 import numpy as np
 import torch
+from functorch._src.partitioners import draw_graph
```

Comment:
Might want to move this import into the function.

Reply:
moved